### PR TITLE
53 review pagination functionality

### PR DESCRIPTION
**Pull request by DominykasSueris**
Aws Logs stream padination functionality fixed, also outline border removed and adjusted to center.

Please approve changes and merge to main @augustinasrce @Konstantin-Bozhkov

### CODE REVIEW

**Comment by DominykasSueris** [comment]
Pagination merge conflict fixed.

**Comment by Konstantin-Bozhkov** [comment]
Please merge branch "main"

File: src/utils/arrays.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const PerPage = 2;
+export const PerPage = 6;
```
Comment (Konstantin-Bozhkov) [inline src/utils/arrays.ts:1]: Lets increase this to 50

**Review by Konstantin-Bozhkov — CHANGES_REQUESTED** [review]

File: src/utils/arrays.ts (hunk 1 above)
Comment (Konstantin-Bozhkov) [reply src/utils/arrays.ts:1]: @DominykasSueris

**Review by Konstantin-Bozhkov — APPROVED** [review]

**Pull request merged**